example/c/include/ICU4XFixedDecimalFormatter.h

```diff
@@ -0,0 +1,36 @@
+#ifndef ICU4XFixedDecimalFormatter_H
```

Comment:
Please add a .gitattributes that marks generated files, not sure what to review.

Reply:
We've elected to not do that because for _diplomat_ the changes to generated files should actually be reviewed.

In this case it's fine if you don't review them because I basically just rewrote everything.

Reply:
For example I don't have to review *every* generated file, but if I want to take a look at them I can. `main.c` on the other hand is handwritten and should be reviewed.

Reply:
Can we handle this separately? I'd like to merge this soon so that we can cut a new release.